src/Behat/Step/Definition.php

```diff
@@ -18,4 +18,5 @@
  */
 interface Definition
 {
+    public function getPattern(): ?string;
```

Comment:
Same here for access to the pattern property, again it is a BC break but I don't expect anyone to have defined other classes that implement this interface